src/index.sass

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
I'd recommend to use `scss` as pre-processor since it's the most popular for now

Reply:
Solved Sr.